index.js

```diff
@@ -5,12 +5,26 @@ const macos = () => execa.stdout('netstat', ['-anv', '-p', 'tcp'])
 	.then(data => Promise.all([data, execa.stdout('netstat', ['-anv', '-p', 'udp'])]))
 	.then(data => data.join('\n'));
 
-const linux = () => execa.stdout('netstat', ['-tunlp']);
+const linux = () => execa.stdout('ss', ['-tunlp']);
 const win32 = () => execa.stdout('netstat', ['-ano']);
 
 const getListFn = process.platform === 'darwin' ? macos : process.platform === 'linux' ? linux : win32;
-const cols = process.platform === 'darwin' ? [3, 8] : process.platform === 'linux' ? [3, 6] : [1, 4];
-const isProtocol = x => /^(tcp|udp)/i.test(x);
+const cols = process.platform === 'darwin' ? [3, 8] : process.platform === 'linux' ? [4, 6] : [1, 4];
+const isProtocol = x => /^\s*(tcp|udp)/i.test(x);
+
+const parsePid = input => {
```

Comment:
Default the parameter to replace the check below: `(input = '') => {`

Reply:
I personally think that the way it is makes the meaning of the code clearer

Reply:
I think enforcing the contract of the the function to accept a string allows the function to be easier to digest. "The best code is the code you don't write" and all that jazz. I suppose it comes down to personal preference in the end.